cockroachdb/README.md

```diff
@@ -5,29 +5,30 @@
 ## Documentation
 
 Below is a brief overview of operating the CockroachDB Helm Chart and some specific implementation details. For additional information on deploying CockroachDB, please see:
+
 > <https://www.cockroachlabs.com/docs/stable/orchestrate-cockroachdb-with-kubernetes.html>
 
 Note that the documentation requires Helm 3.0 or higher.
 
 ## Prerequisites Details
 
-* Kubernetes 1.8
-* PV support on the underlying infrastructure (only if using `storage.persistentVolume`). [Docker for windows hostpath provisioner is not supported](https://github.com/cockroachdb/docs/issues/3184).
-* If you want to secure your cluster to use TLS certificates for all network communication, [Helm must be installed with RBAC privileges](https://helm.sh/docs/topics/rbac/) or else you will get an "attempt to grant extra privileges" error.
+- Kubernetes 1.8
```

Comment:
Is there a motivation for the change from asterisks to dashes in this file?

Reply:
Nope.  Looks like the Prettier extension in my IDE did some auto-formatting.  Reverted the change to only include updates to the version numbers.